features/polyExplorer/src/screens/stories/messengerStory.jsx

```diff
@@ -24,20 +26,65 @@ const MessengerStory = () => {
         "iMessage",
     ];
 
+    const summaryBullets = [
```

Comment:
There seems to be a lot of common code between this one and DigitalGiant. Maybe factor out to a common ancestor class?

Reply:
That is because right now it's basically a copy, wince we haven't started working on the content yet. That will change quite a lot